Jelly/Classes/JellyPresentationController.swift

```diff
@@ -32,7 +32,8 @@ class JellyPresentationController : UIPresentationController {
         switch self.presentation.backgroundStyle {
         case .blur(let effectStyle):
             animateBlurView(effectStyle: effectStyle)
-        case .dimmed:
+        case .dimmed(let alpha):
+            dimmingView.backgroundColor = UIColor(white: 0.0, alpha: alpha)
```

Comment:
I did have it another way where I changed the setupDimmingView function to take the alpha param, and change it there, but it was a little messier having to check if case for dimmed in the init. I can change it back to that before I submit the next PR